src/server/rdb_test.cc

```diff
@@ -760,41 +760,25 @@ TEST_F(RdbTest, LoadHugeList) {
 TEST_F(RdbTest, LoadHugeStream) {
   TEST_current_time_ms = 1000;
 
-  // Add a huge stream (test:0) with 2000 entries, and 4 1k elements per entry
+  // Add a huge stream (test:0) with 2000 entries, and 1-4 elements per entry
   // (note must be more than 512*4kb elements to test partial reads).
-  // We add 2000 entries to the stream to ensure that the stream, because populate strream
-  // adds only a single entry at a time, with multiple elements in it.
-  for (unsigned i = 0; i < 2000; i++) {
-    Run({"debug", "populate", "1", "test", "2000", "rand", "type", "stream", "elements", "4"});
-  }
+  Run({"debug", "populate", "1", "test", "2000", "rand", "type", "stream", "elements", "2000"});
+
   ASSERT_EQ(2000, CheckedInt({"xlen", "test:0"}));
   Run({"XGROUP", "CREATE", "test:0", "grp1", "0"});
   Run({"XGROUP", "CREATE", "test:0", "grp2", "0"});
   Run({"XREADGROUP", "GROUP", "grp1", "Alice", "COUNT", "1", "STREAMS", "test:0", ">"});
   Run({"XREADGROUP", "GROUP", "grp2", "Alice", "COUNT", "1", "STREAMS", "test:0", ">"});
 
-  auto resp = Run({"xinfo", "stream", "test:0"});
-
-  EXPECT_THAT(
```

Comment:
The problem that you are not able to do this checks because the number of fields is random now? I actually do see a lot of value to the random, so we can remove it and do 4 fields in each entry